README.md

```diff
@@ -54,7 +57,9 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+a = residents
+a.delete(Belle)
+guests.add_person (Belle)
 ```
 
```

Comment:
Here's one solution:

```rb
town[:residents].delete("Belle")
town[:castle][:guests] << "Belle"
```

In your code, you need to access the `residents` and the `castle` keys through the `town` object. Then access `guests` through `castle`. There's no `add_person` method here that you created. You could have pushed "Belle" into the `guests` array. Also, "Belle" needs to be a string. 

Testing your code in pry would show you these errors.